File: Models/Clinics/Clinic.cs
using Peohe.Models.Attendances;
using Peohe.Models.Balances;
using Peohe.Models.Doctors;
using System.Collections.Generic;

namespace Peohe.Models.Clinics
{
    public class Clinic
    {
        public int ClinicId { get; set; }
        public string Name { get; set; }
        public ICollection<Balance> Balances { get; set; }
        public double Percentage { get; set; }
        public ICollection<Doctor> Doctors { get; set; }
        public ICollection<Attendance> Attendances { get; set; }
    }
}

File: ClientApp/src/app/models/Clinic.ts
import { Attendance } from "../attendance/attendance";
import { ClinicDoctor } from "./ClinicDoctor";

export class Clinic{
    clinicId: string;
    name: string;
    percentage: number;
    clinicDoctors: ClinicDoctor[];
    attendances: Attendance[];
    deleted?: string;
    aplicationUserId: string;
}